XenCenterLib/EncryptionUtils.cs

```diff
@@ -66,7 +66,7 @@ public static string Protect(string data)
         {
             byte[] dataBytes = Encoding.Unicode.GetBytes(data);
 
-            byte[] protectedBytes = ProtectedData.Protect(dataBytes, GetSalt(), DataProtectionScope.CurrentUser);
+            byte[] protectedBytes = ProtectedData.Protect(dataBytes, GetSalt(), DataProtectionScope.LocalMachine);

```

Comment:
We are currently investigating other ways of passing the server list to the service, without having to modufy the encryption level
For the time being, we can keep these changes, so that we can progress with the implementation. But we plan to revert them later when a different solution is agreed.
Cheng, could you please include this change into a #if directive, so that the change is only in place when compiling for this testing configuration.